src/en/wizden-staff/maintainer/adding-new-maintainers.md

```diff
@@ -0,0 +1,36 @@
+# Adding New Maintainers
+
+This document defines the process for adding new Maintainers to Space Station 14.
+
+## Requirements
+
+Maintainers represent Space Wizards as staff. Certain requirements must be met before a person can become a Maintainer:
+- A vote must be held according to the Voting Process outlined below.
+- The nominated Maintainer must be a member of the Space Station 14 Discord server.
+- If a nominated Maintainer has caused administration issues in the past, these issues must be raised and discussed during the voting process. Administation issues include, but are not limited to, being banned from the game, banned from the Discord server, or having a history of being rude to other members of the community.
+- The nominated Maintainer must have a good understanding of the Space Station 14 codebase and how it works. This must be raised and discussed during the voting process.
+- Have a solid understanding the culture and values of the game and community
+
+### Voting Process
+
+To add a new Maintainer, a current Maintainer has to nominate the candidate. Once a person has been nominated, the following process should be followed:
+1. A vote will be created in the "Internal" category on the Space Station 14 Discourse, using the appropriate template. This vote will be open for one week, and all Maintainers will be notified of the vote.
+2. The vote must be open for 1 week, and all current maintainers should be notified of the vote. During this time, maintainers should discuss the nomination, raise concerns, and ask any questions they may have. If there is a clear majority of Maintainers in favor or against the nomination, the vote may be closed early.
+    - If there is no clear majority, the vote must remain open for the full week.
+    - Concerns about a nominated person must be addressed before the vote may close. This will delay the vote until the concerns are resolved.
+    - For a vote to pass, it must have a supermajority (66%) of votes in favor. 
+3. At the end of the week, the vote will be closed and the results will be announced. If the vote passes, the person being nominated will be contacted by a Lead Maintainer and asked if they would like to accept the position. If they agree, they may then be added to the Maintainers list.
+
+### Special Maintainer Roles
+
+Some roles are special Maintainer roles and follow a different process to the normal Maintainer role. These roles are as follows:
+- **Head Mapper**: A Maintainer responsible for ensuring mapping standards and approving and merging mapping PRs.
+- **Art lead**: A Maintainer responsible for ensuring a consistent art style and approving sprite PRs.
+- **Robust Toolbox Maintainer**: A Maintainer of Robust Toolbox, our underlying game engine. 
+
```

Comment:
I'd probably remove this as I think RT maintainers would entirely be under PJB's purview unless that were no longer feasible, in which case we would add them under the policy.